powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationCreateRequest.java

```diff
@@ -63,6 +45,6 @@ public class OperationCreateRequest {
     private Boolean proximityCheckEnabled;
 
     @Schema(description = "Activation ID. It is possible to specify a single device (otherwise all user's activations are taken into account).", requiredMode = Schema.RequiredMode.NOT_REQUIRED, maxLength = 37)
+    @Size(max = 37, message = "Activation ID must not exceed 37 characters when creating operation")
```

Comment:
It seems the extra arguments `requiredMode` and `maxLength` in `@Schema` annotation are redundant now.